subprojects/rzwinkd/meson.build

```diff
@@ -1,9 +1,6 @@
 project('rzwinkd', 'c',
   license: 'LGPL-3.0-only',
   meson_version: '>=0.55.0',
-  default_options: [
-    'c_std=c99',
-  ]
```

Comment:
define this instead of using `add_project_arguments`

Reply:
AFAIK, this cannot be designed conditional. However, the MSVC-Compiler could get problems with these flags, therefore we need to check `cc.has_argument('--std=*'`. (It was missed here in the else-branch. However, in https://github.com/rizinorg/rizin/pull/4680/files#diff-d1ca507ad4335da90daae84cf012d657f3698491ecf5298a3e230442cf8f4ae1 it has been done as intended)

Reply:
you can always override it c_std like you do when you invoke `default_options`

Reply:
could you maybe explain (maybe in a Review-Comment on https://github.com/rizinorg/rizin/pull/4684), how you think this should look like in detail.
Because `add_project_arguments` is IMHO a completely different feature of meson-build, than it's options. The goal of `add_project_arguments` is, to provide arguments that are added to the project (conditional as well as non-conditional), such as `-D`-predefines. On the other hand, options might be set by the packager (via `meson configure`) and it would be highly non-intuitive to overwrite them by the build-script?